nexus/src/app/sagas/common_storage.rs

```diff
@@ -111,14 +101,40 @@ pub(crate) async fn call_pantry_attach_for_disk(
             )))
         })?;
 
+    info!(
+        log,
+        "sending attach for disk {disk_id} volume {} to {pantry_address}",
```

Comment:
Should this log be in `call_pantry_attach_for_volume` instead, since it does the actual sending?

Reply:
yep! done in 3f10542f3